src/Bicep.Core.IntegrationTests/ScopeTests.cs

```diff
@@ -349,5 +354,43 @@ public void Tenant_scope_resources_can_be_deployed_from_anywhere(string targetSc
                 }
             }
         }
+
+        [TestMethod]
+        public void Existing_resource_with_symbolic_names_enabled_includes_scope_properties()
```

Comment:
>Existing_resource_with_symbolic_names_enabled_includes_scope_properties [](http://example.com/codeflow?start=20&length=71)

I manually validated that this template deploys successfully and produces the right result.